src/cycling/miscHandling.java

```diff
@@ -1,5 +1,177 @@
 package cycling;
 
-public class miscHandling {
+import java.util.HashMap;
+
+public class MiscHandling {
     
+    // Hash of team IDs to their respective team object
+    private HashMap<Integer, Team> teamsHash;
+    // Hash of race IDs to their respective race object
+    private HashMap<Integer, Race> racesHash;
+
+    /**
+     * Adds a team object linked to its ID into a team hashmap
+     * 
+     * @param teamObject The object to be added to the array
+     * 
+     * @throws DuplicatedResultException If the object has already been added
+     */
+    public void add(Team teamObject) throws DuplicatedResultException {
```

Comment:
This might need renaming as "add" is an ass name for a method, espeically public

Reply:
yeah, but it makes it an overloaded function